chapters/data-analysis.tex

```diff
@@ -1,182 +1,237 @@
-They also connects code and folder structure through globals or objects. 
-In short, a master script is a human-readable map to the tasks, 
-files and folder structure that comprise a project.  
-Having a master script eliminates the need for complex instructions to replicate results. 
-Reading the master do-file should be enough for anyone unfamiliar with the project
+They briefly describe what each code does,
+and map the files they require and create.
+They also connect code and folder structure through macros or objects.
+In short, a master script is a human-readable map of the tasks,
+files, and folder structure that comprise a project.
+Having a master script eliminates the need for complex instructions to replicate results.
+Reading it should be enough for anyone unfamiliar with the project
 to understand what are the main tasks, which scripts execute them,
-and where different files can be found in the project folder. 
+and where different files can be found in the project folder.
 That is, it should contain all the information needed to interact with a project's data work.
 
-\subsection{Version control}
-Finally, everything that can be version-controlled should be. 
+\subsection{Implementing version control}
+
+Establishing a version control system is an incredibly useful
+and important step for documentation, collaboration and conflict-solving.
 Version control allows you to effectively track code edits,
-including the addition and deletion of files. 
-This way you can delete code you no longer need, 
+including the addition and deletion of files.
+This way you can delete code you no longer need,
 and still recover it easily if you ever need to get back previous work.
+Everything that can be version-controlled should be.
 Both analysis results and data sets will change with the code.
-You should have each of them stored with the code that created it.
-If you are writing code in Git/GitHub,
+Whenever possible, you should track have each of them with the code that created it.
+If you are writing code in Git or GitHub,
 you can output plain text files such as \texttt{.tex} tables
 and metadata saved in \texttt{.txt} or \texttt{.csv} to that directory.
 Binary files that compile the tables,
 as well as the complete data sets, on the other hand,
-should be stored in your team's shared folder. 
+should be stored in your team's shared folder.
 Whenever data cleaning or data construction codes are edited,
 use the master script to run all the code for your project.
-Git will highlight the changes that were in data sets and results that they entail. 
+Git will highlight the changes that were in data sets and results that they entail.
 
 %------------------------------------------------
 
-\section{Data cleaning}
+\section{De-identifying research data}
+
+The starting point for all tasks described in this chapter is the raw data
+which should contain only materials that are received directly from the field.
```

Comment:
```suggestion
which should contain only information that are received directly from the field.
```

We never use materials in this context anywhere else? is there a good reason for that?